### PR TITLE
latest changelog since 1.2.0b1

### PR DESCRIPTION
**Pull request by minrk**
@consideRatio I think we can release 1.2.0 final with this

### CODE REVIEW

**Comment by consideRatio** [comment]
@minrk I added a commit where I moved 4 PRs so they were sorted like the other PRs by creation date.

This LGTM! I think everything looks good in this changelog and jupyterhub's state in general for a 1.2.0 release!

File: docs/source/changelog.md [hunk 1]
```diff
@@ -9,8 +9,6 @@ command line for details.
 
 ## 1.2
 
-### 1.2.0b1
-
 JupyterHub 1.2 is an incremental release with lots of small improvements.
```
Comment (consideRatio) [inline docs/source/changelog.md:12]:
```suggestion
### [1.2.0] 2020-10-29

JupyterHub 1.2 is an incremental release with lots of small improvements.
```

Comment (consideRatio) [reply docs/source/changelog.md:12]: This would make the formatting align with other releases formatting.

**Comment by minrk** [comment]
Great! I'll cut 1.2.0 right now

**Comment by minrk** [comment]
Published!

**Comment by consideRatio** [comment]
I'm updating z2jh - perhaps publish 0.10.0 straight away as well? I'd prefer to add 0.10.1 later than iterate with more beta's without clear hope of feedback.

**Comment by consideRatio** [comment]
PR to bump: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/1885

**Comment by minrk** [comment]
Yes, I think that's sensible

**Pull request merged**